standalone/portal-api/src/modules/buyer/buyers.controller.ts

```diff
@@ -0,0 +1,50 @@
+import { ApiTags } from "@nestjs/swagger";
+import { BuyerService } from "./services/buyer.service";
+import { Body, Controller, Get, Param, Patch, Post } from "@nestjs/common";
+import { GetBuyerResponse } from "./dto/response/get-buyer-response.dto";
+import { CreateBuyerRequest } from "./dto/request/create-buyer.dto";
+import { UpdateBuyerRequest } from "./dto/request/update-buyer.dto";
+
+@ApiTags('Buyers')
+@Controller({
+  path: 'buyers',
+  version: '1',
+})
+export class BuyersEntityController {
+  constructor(private readonly buyerService: BuyerService) {
+
+  }
+
+  @Post()
+  async createEntity(
+    @Body() entity: CreateBuyerRequest,
+  ): Promise<GetBuyerResponse> {
+    return await this.buyerService.createEntity(entity);
+  }
+
+  @Post('/:id')
+  async updateEntity(
+    @Param('id') id: string,
```

Comment:
```suggestion
  @PUT('/:id')
  async updateEntity(
    @Param('id') id: string,
```

Is the use of `POST` method here intentional or did you mean to use `PUT`

Reply:
I meant put :) thank you